src/app/component/SearchOverlay/SearchOverlay.style.scss

```diff
@@ -39,6 +39,8 @@
     }
 
     &_isVisible {
+        z-index: 100;
```

Comment:
This z-index I think is already used by some other elements, ensure they do not conflict.

Reply:
does not overlap demo notice and navigation tabs. Did not test the install prompt, though.